docs/introduction/Car-registry-tutorial.rst

```diff
@@ -2,73 +2,76 @@
 |                     |                                                                                                                                                                                                                       | year -- vehicle year production                                                                                                                                                                                     |
 |                     |                                                                                                                                                                                                                       | model -- car model                                                                                                                                                                                                  |
 |                     |                                                                                                                                                                                                                       | color -- car color                                                                                                                                                                                                  |
 +---------------------+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
-| CarSellOrderBox     | Box which contains car sell order data, which could be stored and operated in Sidechain.                                                                                                                              | boxData -- contains  car sell order data                                                                                                                                                                            |
+| CarSellOrderBox     | Box which contains car sell order data, which could be stored and operated in the Sidechain. ("...which is stored and managed..."?)                                                                                                                             | boxData -- contains  car sell order data                                                                                                                                                                            |
```

Comment:
| CarSellOrderBox     | is a box which contains CarSellOrder data which could be stored and operated in the sidechain.                                                                                                                            | boxData -- contains CarSellOrder data                                                                                                                                                                            |